app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.errors import VaccineError, NotWearingMaskError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
It's a good practice to use type annotations for all function arguments and the return type. Here, you should specify the type of elements in the `friends` list.